developer_setup/debugging.md

```diff
@@ -0,0 +1,64 @@
+## Debugging ManageIQ with Remote Pry
+
+Interactive debugging using pry or byebug is normally difficult with ManageIQ, as `rake evm:start` launches the server and worker processes in the background. Therefore the only way to do interactive debugging is with `pry-remote` or `byebug` running in remote mode.
+
+Running ManageIQ with `byebug` was exceedingly slow, so this guide will focus on remote debugging using `pry`.
+
+### Adding dependencies
+
+You'll need to add the following gems to your `Gemfile`:
```

Comment:
I second adding this to `bundler.d` dir. It's also in line with the [recent change](https://github.com/ManageIQ/manageiq/commit/3a19899cef6977eeaf737517ba4b80750d283bcb).